core/src/test/java/org/apache/calcite/test/catalog/MockCatalogReaderSimple.java

```diff
@@ -155,6 +155,8 @@ public MockCatalogReaderSimple(RelDataTypeFactory typeFactory,
     deptNestedTable.addColumn("NAME", fixture.varchar10Type);
     deptNestedTable.addColumn("SKILL", fixture.skillRecordType);
     deptNestedTable.addColumn("EMPLOYEES", fixture.empListType);
+    deptNestedTable.addColumn("ADMINS", fixture.varchar5ArrayType);
+    deptNestedTable.addColumn("OFFICES", fixture.rectilinearPeekCoordMultisetType);
```

Comment:
Do not modify existing table/schema to avoid introducing unnecessary change of plans.

Reply:
I am really confused about your review.

What do you mean by 

> You should add a new nested StructKind.PEEK_FIELDS type and write test with that.

Right now you again think we should create a new table/schema to avoid it. I guess it's expected for you to know that this table is used in a lot of places. Maybe I am wrong here?

And what's the drawbacks of so? It's test fixtures and should be shared with multiple tests. Do we need to create a new test fixtures for every PR if it changes other test assertions?

If you insist, please think it again and let me know if I create another table with something like
`  deptNestedExtended.addColumn("NAME", fixture.varchar10Type);
    deptNestedExtended.addColumn("ADMINS", fixture.varchar5ArrayType);
    deptNestedExtended.addColumn("EMPLOYEES", fixture.empListType);
    deptNestedExtended.addColumn("OFFICES", fixture.rectilinearPeekCoordMultisetType);
`

Reply:
-1 because we should never break existing test because of not relevant change.

I don’t care if you reuse the test code base but at least let’s not break other tests.

Reply:
OK. I will add a new table and add above columns in it and use that for tests.

Reply:
Done